extensions/wikia/Discussions/tests/SpecialForumRedirectControllerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @group Integration
+ */
+class SpecialForumRedirectControllerTest extends WikiaDatabaseTest {
+	const THREAD_ID = 1;
+	const REPLY_ID = 2;
+
+	protected function setUp() {
+		parent::setUp();
+		require_once __DIR__ . '/../controllers/SpecialForumRedirectController.class.php';
+	}
+
+	/**
+	 * @dataProvider provideGetRedirectableForumTitle
+	 * @param int $articleId
+	 * @param int $expectedArticleId
+	 */
+	public function testGetRedirectableForumTitle(
+		int $articleId, int $expectedArticleId
+	) {
+		$article = Article::newFromID( $articleId );
```

Comment:
Figure out how to test the `https://tardistest.wikia.com/wiki/Thread:3587` case.